src/api/gitHub.js

```diff
@@ -290,11 +221,114 @@ export const determineGraphQLEndpoint = async (server) => {
   }
 };
 
+
+/**
+ * fetch options for fetchRepoInOrg
+ *
+ * @param {string} org the organization
+ * @param {string} token the token
+ * @param {string} cursor the last repository fetched
+ * @param {boolean} selfsigned the selfsigned option 
+ * @returns {object} the fetch options
+ */
+export const fetchRepoInOrgInfoOptions = async (org, token, cursor, selfsigned) => {
```

Comment:
This doesn't need to be `async` because it doesn't use promises anywhere. You can remove this, and then avoid the `await` when you call the function:

```suggestion
export const fetchRepoInOrgInfoOptions = (org, token, cursor, selfsigned) => {
```